cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,124 @@
 /// <reference types='cypress' />
+Cypress.on('uncaught:exception', (_err) => {
+  return false;
+});
+
+let user;
 
 describe('Web Tables page', () => {
-  it('', () => {
+  before(() => {
+    // eslint-disable-next-line arrow-parens
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('Should check pagination and selector of rows.', () => {
+    cy.get('.-previous > .-btn').should('contain', 'Previous');
+    cy.get('.-next > .-btn').should('contain', 'Next');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+    cy.get('select').select('5 rows');
+    cy.get('select').should('contain', '5 rows');
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
```

Comment:
for adding new user, create new method addNewUser() to avoid code duplication

Reply:
Changes in commit https://github.com/mate-academy/qa_cypress_e2e_web_tables/pull/188/commits/a38dbac156617e5fbc07bf1a11770c0082a88050